<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideBarDemo</class>
 <widget class="QWidget" name="SideBarDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SideBarDemo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget.SideBar {
    border-right: 1px solid rgb(170, 170, 170);
    background: qlineargradient(spread:reflect,
        x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(50, 50, 50, 255),
        stop:1 rgba(150, 150, 150, 255));
}

QWidget.SideBar QToolButton {
    font-size: 13px;
    font-weight: normal;
    border: none;
    color: rgb(240,240,240);

    padding-left:12px;
    padding-right: 12px;
    padding-top: 5px;

    border-top: 1px dashed rgba(0,0,0,0);
    border-bottom: 1px dashed rgba(0,0,0,0);
    border-right: 1px dashed rgba(0,0,0,0);
}

QWidget.SideBar QToolButton:hover {
    color: rgb(200, 200, 200);
}

QWidget.SideBar QToolButton[current=&quot;true&quot;] {
    border-top: 1px solid rgb(90,90,90);
    border-right: 1px solid rgb(200, 200, 200, 200);
    border-bottom: 1px solid rgb(90,90,90);
    color: black;
    background: qlineargradient(spread:pad,
        x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(130, 130, 130, 255),
        stop:1 rgba(230, 230, 230, 255));
}

QWidget.SideBar QToolButton[first=&quot;true&quot;] {
    border-top: 1px dashed rgba(0, 0, 0, 0);
}

QWidget.SideBar .Separator {
    border: none;
    background: qlineargradient(spread:reflect,
        x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(10, 10, 10, 255),
        stop:1 rgba(220, 220, 220, 255));
}

#content {
	border: none;
	border-left: 1px solid gray;
	background: rgba(0, 255, 0, 100);
}</string>
  </property>
  <property name="class" stdset="0">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="sideBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="class" stdset="0">
      <string>SideBar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QToolButton" name="toolButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/resources/button-4.png</normaloff>:/images/resources/button-4.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="toolButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/resources/button-5.png</normaloff>:/images/resources/button-5.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QToolButton" name="toolButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/resources/button-3.png</normaloff>:/images/resources/button-3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="toolButton_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/resources/button-2.png</normaloff>:/images/resources/button-2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="class" stdset="0">
         <string>Separator</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="content">
     <property name="text">
      <string>中心内容区域</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
